tools/commands/test.py

```diff
@@ -0,0 +1,47 @@
+import logging
```

Comment:
Just out of curiosity, why do we need this command at all now that prplmesh_utils.sh provides all this?